db/migrate/20230222155438_create_likes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :authorId
+      t.integer :postId
+      t.text :bio
+      t.datetime :updatedAt
+      t.datetime :createdAt
```

Comment:
- [ ] Kindly delete these columns, these are not required because 👉🏽 `t.timestamps` will crate updated_at and created_at columns